ntpq/Makefile.am: add NTP_HARD_CFLAGS

Pass NTP_HARD_CFLAGS when building ntpq, like in all other ntp
modules, to make sure -fPIC is passed.

Originally taken from
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=494143c3b4921a5c8b8596d58f2c8b98296bf688.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/ntpq/Makefile.am
===================================================================
--- a/ntpq/Makefile.am
+++ b/ntpq/Makefile.am
@@ -23,7 +23,7 @@
 ntpq_LDADD += $(LDADD_NTP)
 noinst_HEADERS=	ntpq.h
 noinst_LIBRARIES=	libntpq.a
-libntpq_a_CFLAGS=	-DNO_MAIN_ALLOWED -DBUILD_AS_LIB
+libntpq_a_CFLAGS=	$(NTP_HARD_CFLAGS) -DNO_MAIN_ALLOWED -DBUILD_AS_LIB
 CLEANFILES=
 DISTCLEANFILES=	.version version.c config.log $(man_MANS)
 ETAGS_ARGS=	Makefile.am
